app/courses/[courseId]/[badgeId]/page.module.css

```
@@ -62,264 +62,308 @@
   justify-content: space-between;
   gap: 1rem;
-  margin-bottom: 2rem;
+  padding-bottom: 0.7rem;
+  margin-bottom: 1.5rem;
+  border-bottom: 2px solid #e3e6ea;
 }
 
 .cardTitle {
   margin: 0;
   color: #000000;
   font-family: 'Lato', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Lato" (font-family-name-quotes)

(font-family-name-quotes)